src/components/widgets/LabsSummary.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { PureComponent } from 'react';
+import * as R from 'ramda';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+import constantsRest from '../../rest/constantsRest';
+import orderRest from '../../rest/orderRest';
+import encounterRest from '../../rest/encounterRest';
+import conceptRest from '../../rest/conceptRest';
+
+export class LabsSummary extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      labResults: []
+    };
+
+    this.fetchLabTestResults = this.fetchLabTestResults.bind(this);
+  }
+
+  componentDidMount() {
+    // PatientID goes here
+    this.fetchLabTestResults('70c9de3d-ce33-420b-818b-332acbfaf776');
+  }
+
+  getConceptValue = (conceptValue) => {
+    if (typeof conceptValue === 'object' && conceptValue !== null) {
+      return conceptValue.display;
+    } else {
+      return conceptValue;
+    }
+  } 
+
+  async fetchLabTestResults(patientUUID) {
+    // Fetch global properties
+    const encounterTypeResponse = await constantsRest.fetchLabResultsEncounterType();
+    const labResultsTestOrderTypeResponse = await constantsRest.fetchLabResultsTestOrderType();
+    const labResultsTestOrderNumberConceptResponse = await constantsRest.fetchLabResultsTestOrderNumberConcept();
+    const labResultsDateConceptResponse = await constantsRest.fetchLabResultsDateConcept();
+    const labResultsTestLocationQuestionResponse = await constantsRest.fetchLabResultsTestLocationQuestion();
+
+    if (encounterTypeResponse && labResultsTestOrderTypeResponse) {
+      const labResultsTestOrderType = labResultsTestOrderTypeResponse.results[0].value;
+      const encounterTypeUUID = encounterTypeResponse.results[0].value;
+      const labResultsTestOrderNumberConcept = labResultsTestOrderNumberConceptResponse.results[0].value;
+      const labResultsDateConcept = labResultsDateConceptResponse.results[0].value;
+      const labResultsTestLocationQuestion = labResultsTestLocationQuestionResponse.results[0].value;
+
+      // Fetch patient's active orders and encounters
+      const patientOrdersResponse = await orderRest.fetchActiveOrdersByPatient(patientUUID, labResultsTestOrderType);
+      const patientEncountersResponse = await encounterRest.fetchEncountersByPatient(patientUUID, encounterTypeUUID);
```

Comment:
@jomadoye, Dave and discussed this yesterday and based on our updated comments  I think we only need to fetch encounters on this page (not orders), because I think we can get all the information we need from it.  If you look at the comment yesterday on the ticket we are only going to show "reported" results, so we don't need to fetch orders because we won't be displaying order with no results.